website/docs/patterns/objects.md

```diff
@@ -3,96 +3,6 @@ id: objects
 title: Creating objects and calling functions
 sidebar_label: Creating objects and calling functions
 ---
-### Instantiating objects and calling methods and functions
-Use `hydra.utils.call()` (or its alias `hydra.utils.instantiate()`) to instantiate objects, call functions and call class methods. While `instantiate` is an alias for `call`, you may prefer to use `call` for invoking functions and class methods, and `instantiate` for creating objects.
-
-```python
-def call(config: Union[ObjectConf, DictConfig], *args: Any, **kwargs: Any) -> Any:
-    """
-    :param config: An ObjectConf or DictConfig describing what to call and what params to use
-    :param args: optional positional parameters pass-through
-    :param kwargs: optional named parameters pass-through
-    :return: the return value from the specified class or method
-    """
-```
-
-### ObjectConf definition
-ObjectConf is defined in `hydra.types.ObjectConf`:
-```python
-@dataclass
-class ObjectConf(Dict[str, Any]):
-    # class, class method or function name
-    target: str = MISSING
-    # parameters to pass to target when calling it
-    params: Any = field(default_factory=dict)
-```
-
-### Example config node
-```yaml
-# target class name, function name or class method fully qualified name
-target: foo.Bar
-# optional parameters dictionary to pass when calling the target
-params:
-  x: 10
-```
-
-
-#### Example usage
-
-models.py
-```python
-class Foo:
-  def __init__(self, x: int, y: int) -> None:
-    self.x = x
-    self.y = y
-
-  @classmethod
-  def class_method(self, z: int) -> Any:
-    return self(z, 10)
-
-  @staticmethod
-  def static_method(z: int) -> int:
-    return z + 1
-
-def bar(z: int) -> int:
-  return z + 2
-```
-config.yaml
-```yaml
-myobject:
-  target: models.Foo
-  params:
-    x: 10
-    y: 20
-    
-myclassmethod:
-  target: models.Foo.class_method
-  params:
-    z: 5
-
-mystaticmethod:
-  target: models.Foo.static_method
-  params:
-    z: 15
-
-myfunction:
-  target: models.bar
-  params:
-    z: 15
-```
-
-Now to test these instantiate / call them as follows:
-```python
-import hydra
-
-@hydra.main(config_path="config.yaml")
-def app(cfg):
-  foo1: Foo = hydra.utils.call(cfg.myobject)  # Foo(10, 20)
-  foo2: Foo = hydra.utils.call(cfg.myclassmethod)  # Foo(5, 10)
-  ret1: int = hydra.utils.call(cfg.mystaticmethod)  # 16
-  ret2: int = hydra.utils.call(cfg.myfunction)  # 17
-```
-### Real World Example
 One of the best ways to drive different behavior in the application is to instantiate different implementations of an interface.
```

Comment:
Please make a critical pass over this doc to see that it still makes sense.

Reply:
Yeah - sorry I thought I had marked the PR as draft - its not ready for review.